bigbluebutton-html5/private/config/settings.yml

```diff
@@ -312,6 +312,13 @@ public:
   #     url: https://<your-Host>/plugins/SamplePresentationToolbarPlugin.js
   externalVideoPlayer:
     enabled: true
+    # A list of formats supported. Note that adding a new format here does not result in adding support for it in the system
```

Comment:
```suggestion
    # A list of supported formats. Adding a new one will make the client accept it, 
    # but it may not work as it's untested and might not be supported by all browsers
```

what do you think?